### ISSUE TITLE
Name of resource missing from "How to Cite" for collections, commas visible even when text missing

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/fairsharing.github.io#1140]
All collections in the rebuild are missing part of their "How to Cite" text. Here's an example from I'm at https://preview.fairsharing.org/#/3574 :
![cite](https://user-images.githubusercontent.com/143586/125261079-d9df7800-e2f8-11eb-92d1-a64551974296.png)

In contrast, here is a database record (PANGAEA) showing the full "How to Cite" text https://preview.fairsharing.org/#/FAIRsharing.6yw6cp:
![cite2](https://user-images.githubusercontent.com/143586/125261172-ec59b180-e2f8-11eb-8e17-b9f6b446e43c.png)

Collections will never have DOIs, and so the result "How to Cite" will in fact look more like this record  (please note the issue with the commas) https://preview.fairsharing.org/#/2982 for WRMC-BSRN:
![cite3](https://user-images.githubusercontent.com/143586/125261380-1ca15000-e2f9-11eb-9d56-6cb2bd94a9ae.png)

So in short, here are the changes that should happen:

- [x] Collections need their name/short name string adding to the beginning of the "How To Cite" text, in the same format as other record types
- [x] If records don't have a DOI yet, the commas and spacing needs to be correct (in the example above, it should read "FAIRsharing.org:WRMC-BSRN;World Radiation Monitoring Center - Baseline Surface Radiation Network,Last Accessed:Monday, July 12th 2021, 9:29" instead of "FAIRsharing.org:WRMC-BSRN;World Radiation Monitoring Center - Baseline Surface Radiation Network,,Last Accessed:Monday, July 12th 2021, 9:29" (notice the extra comma has been removed)
